src/irmin-pack/layout.ml

```diff
@@ -135,11 +139,13 @@ module Classification = struct
   end
 
   module Volume = struct
-    type t = [ `Mapping | `Data | `Control | `Unknown ] [@@deriving irmin]
+    type t = [ `Mapping | `Data | `Control | `Control_tmp | `Unknown ]
```

Comment:
I'm working on the volume cleanup code for when a GC crashes and realized I needed the classification of the temporary file. Since you are not using this in your PR, can you remove `Control_tmp` from this PR and I will add it in mine? (as an aside: it needs to be `Control_tmp of int` to allow for generation comparison)